# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/udev/render-video-group.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- systemd-256.2/rules.d/50-udev-default.rules.in.vanilla	2024-07-12 11:55:15.254058830 +0200
+++ systemd-256.2/rules.d/50-udev-default.rules.in	2024-07-12 11:58:36.137057945 +0200
@@ -50,9 +50,9 @@
 SUBSYSTEM=="media", GROUP="video"
 SUBSYSTEM=="cec", GROUP="video"
 
-SUBSYSTEM=="drm", KERNEL=="renderD*", GROUP="render", MODE="{{GROUP_RENDER_MODE}}"
-SUBSYSTEM=="kfd", GROUP="render", MODE="{{GROUP_RENDER_MODE}}"
-SUBSYSTEM=="accel", GROUP="render", MODE="{{GROUP_RENDER_MODE}}"
+SUBSYSTEM=="drm", KERNEL=="renderD*", GROUP="video", MODE="{{GROUP_RENDER_MODE}}"
+SUBSYSTEM=="kfd", GROUP="video", MODE="{{GROUP_RENDER_MODE}}"
+SUBSYSTEM=="accel", GROUP="video", MODE="{{GROUP_RENDER_MODE}}"
 
 SUBSYSTEM=="misc", KERNEL=="sgx_enclave", GROUP="sgx", MODE="0660"
 SUBSYSTEM=="misc", KERNEL=="sgx_vepc", GROUP="sgx", MODE="0660"
